src/lib/filters.ts

```diff
@@ -41,7 +41,7 @@ export function isClusterFilter(
     : undefined;
 }
 
-/** Return an expression that returns a value based on whether the feature is hovered. */
+/** Return an expression that returns a value based on whether the feature is hovered. Requires manageHoverState to be enabled for the layer. */
```

Comment:
afaict, this repo doesn't use any kind of docstring system, so there's not a way to formally link to another prop/function somewhere, right?

Reply:
Not that I know of.... if you have some suggestions on how to make this work I'd love to hear it :)

Reply:
I'm still new enough to the web space that the lack of standardized doc tooling is confusing :)  I personally think the example code + reading the simple source code is fine as a user, but I'll keep my eyes open and learn how to do this eventually